### PR TITLE
Prefix Hammer command with dollar sign

### PR DESCRIPTION
**Pull request by maximiliankolb**
#### What changes are you introducing?

follow up to https://github.com/theforeman/foreman-documentation/pull/3444/files#diff-8473cc38e67814ebd36b68d2f1f497f3b6813895b19c2270ef78787275685bc8R57

#### Why are you introducing these changes? (Explanation, links to references, issues, etc.)

you can run Hammer CLI as normal user.

#### Checklists

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.13/Katello 4.15
* [ ] Foreman 3.12/Katello 4.14 (Satellite 6.16)
* [ ] Foreman 3.11/Katello 4.13 (orcharhino 6.11 on EL8 only)
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9/6.10)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

File: guides/common/modules/snip_managing-packages-api.adoc [hunk 1]
```diff
@@ -54,7 +54,7 @@ Example API request:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-curl https://_{foreman-example-com}_/api/job_invocations \
+$ curl https://_{foreman-example-com}_/api/job_invocations \
```
Comment (rh-max) [inline guides/common/modules/snip_managing-packages-api.adoc:57]: LGTM, thanks Max!

**Review by rh-max — APPROVED** [review]
LGTM, thank you.

**Comment by maximiliankolb** [comment]
Setting "tech review done" after I've also just merged https://github.com/theforeman/foreman-documentation/commit/1137d3a971688a8bfef350962fdb1bc9afc17cae

**Comment by maximiliankolb** [comment]
Merged to "master" and cherry-picked to "3.13":
   ce4dea0489..ccd7084b3d  3.13 -> 3.13

**Pull request merged**